examples/flows/evaluation/eval-single-turn-metrics/calculate_context_recall.py

```diff
@@ -0,0 +1,30 @@
+from promptflow import tool
+import json
+import numpy as np
+
+
+@tool
+def calculate(llm_result: str) -> str:
+    try:
+        score = 1
+        response = json.loads(llm_result)
+        if response:
+            result = response.get("result", "")
+            print(result)
+            if result:
+                response = [
+                    int(item.get("attributed", "").lower() == "yes" or item.get("attribited", "").lower() == "yes")
```

Comment:
attribited looks typo? Why need to handle?

Reply:
Better to add a comment here? Which may help other devs modify this in the future.

Reply:
Added.